_posts/2019-02-05-canvas-spheres.html

```diff
@@ -0,0 +1,112 @@
+---
+title: Canvas spheres
+thumbnail: -SITE_URL-/images/canvas-spheres.jpg
+excerpt: "An experiment with HTML5 canvas, requestAnimationFrame and ESLint."
+# githublink: "#"
+bitbucketlink: "https://bitbucket.org/Dries_Sercu/vanilla-js-canvas-spheres"
+demolink: "-SITE_URL-/projects/vanilla-canvas-spheres/index.html"
+---
+<p>
+	Inspired by a bed sheet from Ikea I wanted to make a small app that copied the look. You can use it as a screensaver too!
```

Comment:
Inspired by a bed sheet from IKEA, I wanted to make a small application that copied the look. You can use it as a screensaver too!